openapi/src/main/java/io/helidon/openapi/Serializer.java

```diff
@@ -111,6 +111,7 @@ static class CustomRepresenter extends Representer {
         CustomRepresenter(Map<Class<?>, ExpandedTypeDescription> types,
                 Map<Class<?>, ExpandedTypeDescription> implsToTypes, DumperOptions dumperOptions,
                 DumperOptions.ScalarStyle stringStyle) {
+            super(dumperOptions);
             this.implsToTypes = implsToTypes;
             this.dumperOptions = dumperOptions;
```

Comment:
(Do you still need this?)

Reply:
Yes. The no-args constructor in the superclass was removed.

Reply:
Right, that's what I meant (the `private` field).